Woof/Woof/ViewModel/OwnerMainTabViewModel.swift

```diff
@@ -0,0 +1,9 @@
+import Foundation
+
+/// The view model for a owner main tab view, responsible for preparing and processing data for the view.
+final class OwnerMainTabViewModel: ObservableObject {
+    /// Resets the current owner role  in the app by default value `none`.
+    func resertCurrentOwnerRole() {
```

Comment:
```suggestion
    func resetsCurrentOwnerRole() {
```

?

Could you please enable spell checking and/or install external spell checker?

Reply:
Also we use imperative verbs in the names of method.
Please read Swift API guideline carefully.

Reply:
Sorry for this!  The spell checking is enable.

<img width="680" alt="Screenshot 2023-06-13 at 11 21 38 PM" src="https://github.com/ios-course/ironfoudation-team-project/assets/120586844/5165585e-cc68-4fa4-85d0-c9101ecb60ce">